test/unit/org/apache/cassandra/metrics/BufferPoolMetricsTest.java

```diff
@@ -51,21 +52,21 @@ public static void setup() throws ConfigurationException
     @Before
     public void setUp()
     {
-        BufferPool.setMemoryUsageThreshold(16 * 1024L * 1024L);
+        bufferPool.unsafeSetMemoryUsageThreshold(16 * 1024L * 1024L);
     }
 
     @After
     public void cleanUp()
     {
-        BufferPoolTest.resetBufferPool();
+        bufferPool.unsafeReset();
         metrics.misses.mark(metrics.misses.getCount() * -1);
     }
 
     @Test
     public void testMetricsSize()
     {
         // basically want to test changes in the metric being reported as the buffer pool grows - starts at zero
```

Comment:
@jasonstack Should we also test the state of `usedSize` and `overflowSize` here? (We could have a separate test too, but piggybacking on this one seems okay too...)

Reply:
+1